models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -122,8 +122,16 @@ public function getRenderType(): ?string
         return $this->renderType;
     }
 
-    public function setDefaultValue(array $defaultValue): static
+    /**
+     * @return $this
+     */
+    public function setDefaultValue(array|string|null $defaultValue): static
     {
+        if (!$defaultValue) {
+            $defaultValue = null;
+        }elseif (is_string($defaultValue)){
+            $defaultValue = [$defaultValue];
+        }
```

Comment:
Maybe better
```suggestion
        if (is_string($defaultValue)) {
            $defaultValue = $defaultValue !== '' ? [$defaultValue] : null;
        }
```

Reply:
My current project has `null` and `""` and `somestring` values which broke, I would suspect that perhaps some integers could have made it's way into the default value storage as well. Maybe check for scalar values would make it more forgiving?

And for v12.0 create a migration which forces array type for all while making the type stricter again?

Reply:
Where did these defaultValues came from? 
Was the field type a single select in the past? 
So after the convert in the backend the default value from the single selecte still exists? 

If this is the case, I think we could not make it type strict. This can also possible in v12. We should than also check the case when someone convert the multi select in a single select.

Reply:
They were created using v6 or v10, some could possibly have been selects or other field types at one point as well.